Testing/Testing - Battery Usage.md

```diff
@@ -0,0 +1,69 @@
+# Android - Profiler
+
+This tool helps you find where your app uses more energy than necessary.  
+It consists of three timelines: Event timeline, Energy timeline and System timeline.  
+Profiler monitors use of CPU, network, GPS and system events and presents them in form of bars and horizontal lines (system events).  
```

Comment:
Provuci pliz i kroz Grammarly tekst jer ima 10-ak mjesta gdje bi trebali biti članovi "the"/"a". Označit će dosta mjesta, ali ne treba na svima. Ako ćeš trebati pomoć, samo javi :)